### DISCUSSION TITLE
when making a solution to a problem a padding to disable MSIE and Chrome friendly error page

### DISCUSSION THREAD

**Discussion by edwardtich1** [frenck/spook#569] [Q&A]
[
![2024-01-17_18-57-29](https://github.com/frenck/spook/assets/68006829/91f0ef1a-8658-4d22-b0e1-170f16049bfa)
](url)

`Ошибка
Не удалось загрузить мастер настройки: <html> <head><title>504
Gateway Time-out</title></head> <body> <center><h1>504 Gateway Time-out</h1>‹/center> <hr><center>Web server</center> </body> </html> ‹!-- a padding to disable MSIE and Chrome friendly error page ->   `

**Comment by frenck** [comment]
I'm not sure what I am looking at, or what this has to do with Spook?
